scripts/helper/schedule-webworker.js

```diff
@@ -37,27 +37,33 @@ function getEndTime(date, startTime, endTime, totalNumber, number) {
   return result.getHours() + ':' + result.getMinutes();
 }
 
+function hasSpeakersAndSessionsAndSchedule(speakers, sessions, schedule) {
+  return !Object.keys(speakers).length || !Object.keys(sessions).length || !Object.keys(schedule).length;
```

Comment:
Yes, of course... I messed up when I moved back those modification from our gdg website to my repository branch...

I do the modification right now...